File: javaprog/Series.java
import java.util.Scanner;
class Main
{
  public static void main(String[] args)
  {
    int Count, TermCounter = 2, Product = 1, Counter;
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter the number upto which you want to print the series:");
    Count = sc.nextInt();
    System.out.print("The series upto " + Count + " is 1");
    for(TermCounter = 2; Product <= Count; TermCounter++)
    {
      Product = TermCounter;
      for(Counter = 1; Counter < TermCounter; Counter++)
      {
        Product = Product * TermCounter;
      }
      if(Product < Count)
      {
        System.out.printf(", %d", Product);
      }
    }
    System.out.print(".\n"); 
  }
}

File: pythonprog/series.py
print("Enter the number upto which you want to print the series:", end = '')
Count = int(input())
print("The series upto {0} is 1 " .format(Count), end ="")
TermCounter = 2
Product = 1
while(Product <= Count):
  Product = TermCounter
  for Counter in range (1, TermCounter):
    Product = Product * TermCounter
  if(Product < Count):
    print(", {0}".format(Product), end = ' ')
  TermCounter = TermCounter + 1
print('.\n', end = "")  
